src/main/java/org/osc/sdk/controller/api/SdnRedirectionApi.java

```diff
@@ -226,10 +227,11 @@ void updateInspectionHook(InspectionHookElement existingInspectionHook)
      * </p>
      *
      * @param inspectionPort  provides the inspection port to be created
+     * @return an element
```

Comment:
registerInspectionPort is responsible for setting up the redirection target on NSC and Nuage. However, it caries InspectionPortElement which has redirection target, egreess and ingress info.
By moving this to registerNetworkElement, we need to modify NetworkElement interface to provide egress, ingress info.